apps/whale-api/src/e2e.module.ts

```diff
@@ -59,7 +61,13 @@ export async function stopTestingApp (container: MasterNodeRegTestContainer | Te
 
 async function createTestingModule (url: string): Promise<TestingModule> {
   return await Test.createTestingModule({
-    imports: [AppModule.forRoot('memory')]
+    imports: [
+      CacheModule.register(),
+      AppModule.forRoot('memory')
+    ],
+    providers: [
+      DeFiDCache
+    ]
```

Comment:
since we globally export in `apps/whale-api/src/module.api/_module.ts`, we don't need to export this here again?

Reply:
I will test this out on https://github.com/JellyfishSDK/jellyfish/pull/1566

Reply:
```suggestion
    imports: [AppModule.forRoot('memory')]
```